src/components/LmsConfigurations/MoodleIntegrationConfigForm.jsx

```diff
@@ -0,0 +1,339 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
+import { ValidationFormGroup, Input, StatefulButton, Icon } from '@edx/paragon';
+import { snakeCaseFormData } from '../../utils';
+import LmsApiService from '../../data/services/LmsApiService';
+import StatusAlert from '../StatusAlert';
+import NewRelicService from '../../data/services/NewRelicService';
+
+export const REQUIRED_MOODLE_CONFIG_FIELDS = [
+  'moodleBaseUrl',
+  'serviceShortName',
+];
+
+const SUBMIT_STATES = {
```

Comment:
nit: We might want to consider moving these to a constants file, I think we've re-used these a couple of times.

Reply:
Oh yeah they've been used a boatload of times.